src/Controls/src/Core/Accelerator.cs

```diff
@@ -22,13 +21,14 @@ internal Accelerator(string text)
 
 		/// <include file="../../docs/Microsoft.Maui.Controls/Accelerator.xml" path="//Member[@MemberName='Modifiers']/Docs/*" />
 		[EditorBrowsable(EditorBrowsableState.Never)]
-		public IEnumerable<string> Modifiers { get; set; }
+		public IReadOnlyList<string> Modifiers { get; set; }
```

Comment:
Maybe we can just remove the `EditorBrowsable`? In a perfect world, we can change to IReadOnlyList, but is there a real need? We can make Core use `IReadOnlyList` and then use an explicit implementation to convert back and forth:

I am break adverse, but we can if we feel like it is not used too much. Just fears I suppose.

Reply:
Yea, my curiosity is if we can do this

- Leave it as IEnumerable in controls
- get rid of the "set" (This is the part I realize is a breaking change but it was EditorBrowsable false and these APIs do literally nothing right now in NET6 and NET7)
- Make it IReadonly<string> on the interface

Reply:
How would that work? How can I make the interface an IReadOnlyList if we leave the implementation as IEnumerable in Controls? don't they need to be the same type??

Reply:
Explicit interface implementation

IReadOnlyList  IAccelerator.Modifiers => Convert Modifiers to readonly list